packages/dashmate/src/status/statusProviderFactory.js

```diff
@@ -0,0 +1,33 @@
+const getCoreScopeFactory = require('./scopes/core');
```

Comment:
Why do you need this if we have already DI container?

Reply:
This function does not participate in DI